pkg/controller/manage/kmesh_manage.go

```diff
@@ -84,19 +84,17 @@ func NewKmeshManageController(client kubernetes.Interface) (*KmeshManageControll
 				return
 			}
 
-			if !shouldEnroll(pod) {
-				return
-			}
+			if shouldEnroll(pod) || pod.Annotations[KmeshRedirectionAnnotation] == "enabled" {
```

Comment:
better than before though

But could improve a bit more, in case there are pods startup during restart.